test/docfx.RegressionTest/Options.cs

```diff
@@ -11,7 +11,7 @@ internal class Options
         public string Repository { get; set; } = "";
 
         [Option("branch")]
-        public string Branch { get; set; } = "master";
+        public string Branch { get; set; } = "";
```

Comment:
nit: prefer `null` for optional config.
```suggestion
        public string? Branch { get; set; }
```

Reply:
Thought we prefer empty string instead of nullable.
https://github.com/dotnet/docfx/blob/8f581275384d31b250e44c14e1a2c98e3e280325/src/docfx/config/Config.cs#L285